internal/cmd/tailscale/foward_test.go

```diff
@@ -0,0 +1,51 @@
+//go:build testing
+
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+package tailscale
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/daytonaio/daytona/cmd/daytona/config"
+)
+
+func TestFowardPort(t *testing.T) {
```

Comment:
I suggest we remove this test. This should be covered by a functional/integration test. A unit test doesn't do much here.